poster/poster.tex

```diff
@@ -277,7 +277,7 @@
         \begin{alertblock}{Deterministic Optimizing (DO)}
 	\begin{itemize}
 		\item{\large  Optimizes an objective function with set of constraints}
-		\item {\large Replicable}
+		\item {\large Always produces same result}
```

Comment:
There's no reason to leave out "the" here. Without it, the phrase sounds a bit over-contracted.